test-cases/nemesis/longevity-5gb-1h-BootstrapStreamingErrMonkey.yaml

```diff
@@ -0,0 +1,37 @@
+test_duration: 90
```

Comment:
Please rebase, and generate those again

Reply:
@fruch , i rebased. 
How i can generate this file again?

Reply:
`hydra create-nemesis-pipelines`

Reply:
and it moved to other folder, to configurations/nemesis

so please remove this file

Reply:
Removed all files related to one-nemesis jobs